File: CS/EditorsHelper.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;

using DevExpress.XtraEditors;

namespace UseOneStyleController
{
	public class EditorsHelper
	{
		public static void SetStyleController(Control parent, StyleController styleController)
		{
			foreach (Control control in parent.Controls)
			{
				if (control is BaseControl)
				{
					((BaseControl)control).StyleController = styleController;
				}
				SetStyleController(control, styleController);
			}
		}
	}

}

File: VB/EditorsHelper.vb
Imports DevExpress.XtraEditors

Public Class EditorsHelper
    Public Shared Sub SetStyleController(ByVal parent As Control, ByVal styleController As StyleController)
        Dim control As Control
        For Each control In parent.Controls
            If TypeOf control Is BaseControl Then
                CType(control, BaseControl).StyleController = styleController
            End If
            SetStyleController(control, styleController)
        Next
    End Sub
End Class
